var/spack/repos/builtin/packages/icu4c/package.py

```diff
@@ -88,3 +103,34 @@ def configure_args(self):
             args.append("--enable-rpath")
 
         return args
+
+
+class MSBuildBuilder(spack.build_systems.msbuild.MSBuildBuilder, BuildEnvironment):
+    def msbuild_args(self):
+        return [
+            "allinone.sln",
+            self.define("OutputPath", self.spec.prefix),
+            self.define("Configuration", "Release"),
+            self.define("SkipUWP", "true"),
+        ]
+
+    @property
+    def build_directory(self):
+        solution_path = pathlib.Path(self.pkg.stage.source_path)
+        if self.spec.satsifies("@:67"):
+            solution_path = solution_path / "icu"
+        solution_path = solution_path / "source" / "allinone"
+        return str(solution_path)
+
+    def install(self, pkg, spec, prefix):
+        mkdirp(prefix.lib)
+        mkdirp(prefix.bin)
+        mkdirp(prefix.include)
+        mkdirp(prefix.commondata)
```

Comment:
(question) Is it expected `icu4c` will add files to this post-install? I don't see anything being added that directory

Reply:
I was previously, but it was only to support the UWP build, which I chose to disable as we don't use it with Spack (unless we want to add the XBox/hololens as a platform lol)

I'll remove it